workflow-ui/core-android/src/main/java/com/squareup/workflow1/ui/container/ModalScreenOverlayBackButtonHelper.kt

```diff
@@ -16,31 +17,38 @@ import com.squareup.workflow1.ui.onBackPressedDispatcherOwnerOrNull
  * that predates `OnBackPressedDispatcher`.
  */
 @WorkflowUiExperimentalApi
-public fun interface ModalScreenOverlayOnBackPressed {
+public interface ModalScreenOverlayBackButtonHelper {
+  public fun initialize(contentView: View) {
+    // If the content view has no backPressedHandler, add a no-op one to
+    // ensure that the `onBackPressed` call below will not leak up to handlers
+    // that should be blocked by this modal session.
+    if (contentView.backPressedHandler == null) contentView.backPressedHandler = { }
```

Comment:
This is inadequate, can be stomped updates of the content view.

- [ ] wrap the content view factory instead.

Reply:
And this realization is why #805 happened.